slim-seo.php

```diff
@@ -14,8 +14,10 @@
 
 defined( 'ABSPATH' ) || die;
 
+define( 'SLIM_SEO_FILE', __FILE__ );
```

Comment:
I think we don't need this constant, do we?

Reply:
I have used it in the `plugin_actions_links_*` filter:
```
add_filter( 'plugin_action_links_' . plugin_basename(SLIM_SEO_FILE), [...] );
```
Would you still prefer I remove it?

Reply:
Ah I got it!